### PR TITLE
Multiline Command Announcements

### PR DESCRIPTION
**Pull request by SabreML**
## What Does This PR Do

Adds ` as null|message` to the announcement input for Communications Consoles and Requests Consoles, to allow for much better text formatting.

## Why It's Good For The Game

AI and Admin announcements already have this feature, so why not also add it to Command too.

## Images of changes

**Before:**
before

**After:**
Desktop Screenshot 2021 03 01 - 22 05 59 59

Desktop Screenshot 2021 03 01 - 22 06 24 85

## Changelog
:cl:
tweak: Made the announcements for Communications Consoles and Requests Consoles have the same input as AI announcements. (i.e. Multiline announcements)
tweak: Fixed Requests Consoles resetting their announcement input if you exit and try again.
/:cl:

### CODE REVIEW

**Comment by procdrone** [comment]
I never understood why it had to be single line, AI already had multiline announcements. good change.

**Review by Kyep — APPROVED** [review]

File: code/game/machinery/requests_console.dm [hunk 1]
```diff
@@ -193,7 +193,7 @@ GLOBAL_LIST_EMPTY(allRequestConsoles)
 					reset_message(TRUE)
 
 		if("writeAnnouncement")
-			var/new_message = sanitize(input("Write your message:", "Awaiting Input", ""))
```
Comment (AffectedArc07) [inline code/game/machinery/requests_console.dm:196]: Make sure you cant XSS with this please.

**Review by AffectedArc07 — CHANGES_REQUESTED** [review]

File: code/game/machinery/requests_console.dm (hunk 1 above)
Comment (SabreML) [reply code/game/machinery/requests_console.dm:196]: Both of them still get sanitised in the `Announce()` proc here:
https://github.com/ParadiseSS13/Paradise/blob/156d360e57bd57516e934e841e3301d741a5036f/code/defines/procs/announce.dm#L54-L56
As far as I can tell it's all working the same as before.

**Review by AffectedArc07 — APPROVED** [review]
Tested against XSS

**Pull request merged**